### PR TITLE
chore(libs): update blockscout db entities to 6.8.0 state

### PR DESCRIPTION
**Pull request by bragov4ik**

### CODE REVIEW

**Comment by bragov4ik** [comment]
this doesn't work because
- `accounts_tags` table doesn't have primary key (easily fixable by hand here)
- basically [this issue](https://github.com/SeaQL/sea-orm/issues/2342)

so, waiting for the latter to somehow resolve

**Comment by bragov4ik** [comment]
>to specify tables to import
you mean to exclude tables that face the referenced issue?

File: libs/blockscout-db/justfile [hunk 1]
```diff
@@ -27,8 +27,13 @@ generate-entities database_url="postgres://postgres:admin@localhost:5432/blocksc
...
     cargo fmt --all -- --config imports_granularity=Crate
 
+generate-entities-from-blockscout database_url="postgres://postgres:admin@localhost:5432/blockscout":
+    docker run --network=host -e 'DATABASE_URL={{database_url}}' -e 'ECTO_USE_SSL=false' --entrypoint "/bin/sh" --rm blockscout/blockscout:6.8.0 -c 'bin/blockscout eval "Elixir.Explorer.ReleaseTasks.create_and_migrate()"'
+    DATABASE_URL={{database_url}} sea-orm-cli generate entity --lib -o entity/src
+    cargo fmt --all -- --config imports_granularity=Crate
+
 generate-migration database_url="postgres://postgres:admin@localhost:5432/blockscout?ssl=false" migration_file="migration/src/migration.sql":
-    docker run --network=host -e 'DATABASE_URL={{database_url}}' --entrypoint "/bin/sh" --rm blockscout/blockscout:4.1.8 -c 'bin/blockscout eval "Elixir.Explorer.ReleaseTasks.create_and_migrate()"'
+    docker run --network=host -e 'DATABASE_URL={{database_url}}' -e 'ECTO_USE_SSL=false' --entrypoint "/bin/sh" --rm blockscout/blockscout:6.8.0 -c 'bin/blockscout eval "Elixir.Explorer.ReleaseTasks.create_and_migrate()"'
```
Comment (sevenzing) [inline libs/blockscout-db/justfile:36]: move `6.8.0` to `version` variable at the top of justfile please

File: libs/blockscout-db/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "blockscout-db"
-version = "4.1.8"
+version = "5.0.0"
```
Comment (sevenzing) [inline libs/blockscout-db/Cargo.toml:3]: version of crate should be the same as version of blockscout (6.8.0)

File: libs/blockscout-db/entity/Cargo.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "blockscout-db-entity"
-version = "4.1.8"
+version = "5.0.0"
```
Comment (sevenzing) [inline libs/blockscout-db/entity/Cargo.toml:3]: .

File: libs/blockscout-db/migration/Cargo.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "blockscout-db-migration"
-version = "4.1.8"
+version = "5.0.0"
```
Comment (sevenzing) [inline libs/blockscout-db/migration/Cargo.toml:3]: .

File: libs/blockscout-db/Cargo.toml (hunk 2 above)
Comment (bragov4ik) [reply libs/blockscout-db/Cargo.toml:3]: hmmm... what if a breaking change is introduced but blockscout's major version is not changed? i.e. github.com/blockscout/blockscout-rs/pull/1051

Comment (koushiro) [reply libs/blockscout-db/Cargo.toml:3]: how about using `5.0.0+6.8.0` as the version?

Comment (bragov4ik) [reply libs/blockscout-db/Cargo.toml:3]: makes sense, @sevenzing do you agree?

Comment (bragov4ik) [reply libs/blockscout-db/Cargo.toml:3]: we could even do smth like "5.0.0+blockscout-6.8.0" or "5.0.0+blockscout.6.8.0" or "5.0.0+blockscout6.8.0"

**Review by sevenzing — APPROVED** [review]

**Pull request merged**